front/lib/models/assistant/actions/database.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  CreationOptional,
+  DataTypes,
+  InferAttributes,
+  InferCreationAttributes,
+  Model,
+} from "sequelize";
+
+import { front_sequelize } from "@app/lib/databases";
+
+/**
+ * Action Database Configuration
+ */
+export class AgentDatabaseConfiguration extends Model<
```

Comment:
Let's call it `AgentDatabaseQueryConfiguration` maybe to continue with the convention that these things have an action "verb" in their name?